### PR TITLE
Allow manual setting of UI size

### PR DESCRIPTION
**Pull request by entrhopi**
Update to address #229

### CODE REVIEW

File: Blish HUD/GameServices/Graphics/ManualUISize.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+
+namespace Blish_HUD.Graphics {
+    public enum ManualUISize : int {
+        [Description("Automatic - Sync With Game")]
```
Comment (dlamkins) [inline Blish HUD/GameServices/Graphics/ManualUISize.cs:5]: For the sake of consistency, let's just have this read `Sync With Game`.

**Review by dlamkins — CHANGES_REQUESTED** [review]
Let's get the one name change and the rest looks good!

File: Blish HUD/GameServices/Graphics/ManualUISize.cs (hunk 1 above)
Comment (entrhopi) [reply Blish HUD/GameServices/Graphics/ManualUISize.cs:5]: I have copied this from `DPIMethod` should we change it there too then?

Comment (entrhopi) [reply Blish HUD/GameServices/Graphics/ManualUISize.cs:5]: Upon further inspection I'd suggest changing the name of the enum to `UISizeMethod` too?

Comment (dlamkins) [reply Blish HUD/GameServices/Graphics/ManualUISize.cs:5]: I probably am just misremembering things.  I'll check it tonight.  It's probably all fine to leave as is if you copied it over.  :D

**Review by dlamkins — APPROVED** [review]

**Pull request merged**